localstack-core/localstack/services/events/models.py

```diff
@@ -223,6 +227,10 @@ def arn(self) -> Arn:
 EventBusDict = dict[EventBusName, EventBus]
 
 
+ConnectionDict = dict[ConnectionName, DescribeConnectionResponse]
+ApiDestinationDict = dict[ApiDestinationName, DescribeApiDestinationResponse]
```

Comment:
Guess that's something for service owners to judge, but I don't see this blocking for this iteration.

Reply:
This is really a temporary shortcut to get some typing, I would also think this needs another iteration at some point.